src/SSVOpenHexagon/Core/HGScripting.cpp

```diff
@@ -761,6 +761,39 @@ void HexagonGame::initLua_LevelControl()
             "between ``SidesMin`` and ``SidesMax`` inclusively every level "
             "increment.");
 
+    addLuaFn("l_overrideScore", //
+        [this](const std::string& mVar) {
+            try
+            {
+                levelStatus.scoreOverride = mVar;
+                levelStatus.scoreOverridden = true;
+                // Make sure we're not passing in a string
+                lua.executeCode(
+                    "if (type(" + mVar +
+                    ") ~= \"number\") then\n"
+                    "error(\"Score override must be a number value\")\n"
+                    "end");
+            }
+            catch(const std::runtime_error& mError)
```

Comment:
When can this happen, exactly?

Reply:
This should be happening every time the score is being overridden with a different Lua variable. We want to make sure that the variable we are using to override the score has to be some sort of number value. We do not want the score override to be another type like a table, string, boolean, etc. Mainly because those types will be very hard to sort and it will be very arbitrary in the leader boards.